jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+          deprecated: true
+          example: 2103S
+          maxLength: 50
+          description: "The vessel operator-specific identifier of the Voyage.\r\nIn case there are multiple voyages the export voyage is chosen."
+        exportVoyageNumber:
+          type: string
+          x-stoplight:
+            id: s1i146h6hwvac
+          example: 2103S
+          maxLength: 50
+          deprecated: true
+          description: "The identifier of an export voyage. The vessel operator-specific identifier of the export Voyage.\r\n**Deprecated**: Use carrierExportVoyageNumber instead."
+        carrierExportVoyageNumber:
+          type: string
+          x-stoplight:
+            id: t0t9d0navdjro
+          example: 2103S
+          maxLength: 50
+          description: The identifier of an export voyage. The vessel operator-specific identifier of the export Voyage.
+        importVoyageNumber:
+          type: string
+          x-stoplight:
+            id: vrycin3b7q7rt
+          deprecated: true
+          maxLength: 50
+          example: 2103N
+          description: "The identifier of an import voyage. The vessel operator-specific identifier of the import Voyage.\r\n**Deprecated**: Use carrierImportVoyageNumber instead."
+        carrierImportVoyageNumber:
+          type: string
+          x-stoplight:
+            id: oi6th6o9lfmfz
+          description: The identifier of an import voyage. The vessel operator-specific identifier of the import Voyage.
+          maxLength: 50
+          example: 2103N
+        transportCallSequenceNumber:
+          type: integer
+          x-stoplight:
+            id: q1sc106hs3215
+          format: int32
+          example: 2
+          description: Transport operator's key that uniquely identifies each individual call. This key is essential to distinguish between two separate calls at the same location within one voyage.
+        UNLocationCode:
+          type: string
+          x-stoplight:
+            id: xlcunheamxoik
+          deprecated: true
+          example: FRPAR
+          maxLength: 5
+          description: "The UN Location code specifying where the place is located.\r\n\r\n**Deprecated**: Use UN Location Code defined in the location object instead."
+        facilityCode:
+          type: string
+          x-stoplight:
+            id: r9lxafrgjm94r
+          example: ADT
+          maxLength: 6
+          deprecated: true
+          description: "The code used for identifying the specific facility. This code does not include the UN Location Code.\r\n**Deprecated**: Use facilityCode defined in the location object instead."
+        facilityCodeListProvider:
+          type: string
+          x-stoplight:
+            id: rllkwicnjjchj
+          deprecated: true
+          example: SMDG
+          description: "The provider used for identifying the facility Code.\r\n**Deprecated**: Use facilityCodeListProvider defined in the location object instead."
```

Comment:
Even though this is deprecated - it needs to be provided as an enum. Please also modify description with markdown as I have done everywhere else